bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -274,6 +277,7 @@ The channels are kept consistent as much as possible from brand to brand to make
 | `tooBrightAlarm` | Switch (read only) | ONVIF cameras only will reflect the status of the ONVIF event of the same name. |
 | `tooDarkAlarm` | Switch (read only) | ONVIF cameras only will reflect the status of the ONVIF event of the same name. |
 | `pollImage` | Switch | This control can be used to manually start and stop using your CPU to create snapshots from a RTSP source. If you have a snapshot URL setup in the binding, only then can this control can be used to update the Image channel. |
+| `whiteLED` | Dimmer | Turn the visible white LED ON or OFF and if supported dim from 0-100%. |
```

Comment:
Changes look good, but would be nice if this table would have proper markdown, can you fix that?

Reply:
Done but anything further I would prefer to do a full review of the documentation in a separate PR.